src/mock/card-film.js

```diff
@@ -0,0 +1,141 @@
+
+const filmDuration = [
+  '1h 40m',
+  '1h 55m',
+  '2h 30m',
+  '3h 15m',
+];
+
+const description = [
+  'Фрэнк Эбегнейл успел поработать врачом, адвокатом и пилотом на пассажирской авиалинии – и все это до достижения полного совершеннолетия в 21 год. Мастер в обмане и жульничестве, он также обладал искусством подделки документов, что в конечном счете принесло ему миллионы долларов, которые он получил по фальшивым чекам Агент ФБР Карл Хэнрэтти отдал бы все, чтобы схватить Фрэнка и привлечь к ответственности за свои деяния, но Фрэнк всегда опережает его на шаг, заставляя продолжать погоню.',
+  'Повелитель сил тьмы Саурон направляет свою бесчисленную армию под стены Минас-Тирита, крепости Последней Надежды. Он предвкушает близкую победу, но именно это мешает ему заметить две крохотные фигурки — хоббитов, приближающихся к Роковой Горе, где им предстоит уничтожить Кольцо Всевластья.',
+  'Когда засуха, пыльные бури и вымирание растений приводят человечество к продовольственному кризису, коллектив исследователей и учёных отправляется сквозь червоточину (которая предположительно соединяет области пространства-времени через большое расстояние) в путешествие, чтобы превзойти прежние ограничения для космических путешествий человека и найти планету с подходящими для человечества условиями.',
+  'Пострадав в результате несчастного случая, богатый аристократ Филипп нанимает в помощники человека, который менее всего подходит для этой работы, – молодого жителя предместья Дрисса, только что освободившегося из тюрьмы. Несмотря на то, что Филипп прикован к инвалидному креслу, Дриссу удается привнести в размеренную жизнь аристократа дух приключений.',
+];
+
+const postersFilm = [
+  'made-for-each-other.png',
+  'popeye-meets-sinbad.png',
+  'sagebrush-trail.jpg',
+  'santa-claus-conquers-the-martians.jpg',
+  'the-dance-of-life.jpg',
+  'the-great-flamarion.jpg',
+  'the-man-with-the-golden-arm.jpg',
+];
+
+const getRandomItem = (items) => items[getRandomInteger(0, items.length - 1)];
+
+const getRandomItems = (items) => items.filter(() => Boolean(getRandomInteger(0, 1)));
+
+const generatePoster = (posters) => {
+  const randomIndex = getRandomInteger(0, posters.length - 1);
+  return POSTERS_URL + posters[randomIndex];
+};
+
+const generateDateProdaction = () => {
+  const productionYear = [
+    '2019-05-11',
+    '2015-08-16',
+    '1974-03-20',
+    '2001-03-15',
+    '1997-04-09',
+  ];
+
+  const randomIndex = getRandomInteger(0, productionYear.length - 1);
+
+  return new Date(productionYear[randomIndex]);
+};
+
+const generateCardFilm = (id) => ({
+  poster: generatePoster(postersFilm),
+  name: getRandomItem(filmNames),
+  producer: getRandomItem(producers),
+  screenwriters: getRandomItems(writers),
+  actors: getRandomItems(actors),
+  rating: getRandomItem(filmRating),
+  productionYear: generateDateProdaction(),
+  filmDuration: getRandomItem(filmDuration),
+  country: getRandomItem(country),
+  genre: getRandomItems(genre),
+  commentId: id,
```

Comment:
```suggestion
  id,
```
И вынеси это поле на первое место

Reply:
fixed